### DISCUSSION TITLE
Associating user package to Customer link  not customer directly

### DISCUSSION THREAD

**Discussion by pro-cms** [hotspotbilling/phpnuxbill#141] [Ideas]
Hellow there,
I think it would be great to associate each package/Service to Customer Link not customer directly
Why?
Suppose i am running an internate business and i have one customer who has more than 1 location , eg Home,Office1, Office2 that they want the service, For our current approach, the customer will be required to have more than one account for that to be possible. 

Solution
Associating each customer subscription to CustomerLink, which Admin can create a link, and each link will be differentiated by Link Number and any other necessary parameter, Then customer can purchase A specific plan to specific Link number that they want to use.

Also
The current Setup wont be changed, for a personal account, each package./Plan is associated to customer account, For business customers each Plan is associated to Specific link number that they can easly identify

When business login they can have some side menu link "My Links or My service ...." they click there they get  to see a list of service/Links
eg
Link 1 -- active -- expire at 2023-2.2024 -- Some Action
Link 2 -- expired ---expired at 2023--, - Renew ..

Much thanks

**Comment by ibnux** [comment]
Currently
1 customer can have multiple plan
- 1 pppoe plan prepaid
- 1 hotspot plan prepaid
- 1 pppoe plan postpaid
- 1 hotspot plan postpaid

4 plan In 1 router.

And can have multiple plan for multiple router.

If there a 4 router, then it can have 4x4 plan.

Except for radius, only can 4.

But for your ideas, maybe customer can have multiple same plan? But for different locations?

**Reply by pro-cms** [reply]
Yes sure, If customer have more than one service ,eg we need all to be either hotspot or ppoe, eg
1. Office internet 
2. Home internet 
3. Propert X internet 

I have same scenario and since I intend to fully move to phpnux billing  , this feature will make the platform robust but also One account with all user service
